src/live_restore/live_restore_fs.c

```diff
@@ -1229,6 +1229,55 @@ __wt_live_restore_fh_to_metadata(WT_SESSION_IMPL *session, WT_FILE_HANDLE *fh, W
     return (ret);
 }
 
+/*
+ * __wt_live_restore_clean_metadata_string --
+ *     Live restore stores bitmap information in the metadata file and uses -1 to indicate the file
+ *     has been fully migrated. This value should be cleaned up when live restore completes, but we
+ *     can only do so if the file is open when we force a checkpoint during the clean up phase, or
+ *     if its btree is dirtied after live restore completes. If neither of these cases happen we
+ *     don't update the metadata on disk and the -1 value persists. A future live restore could see
+ *     this value and incorrectly assume the file has been migrated. To prevent this manually remove
+ *     the -1 value when taking a backup.
+ */
+void
+__wt_live_restore_clean_metadata_string(WT_SESSION_IMPL *session, const char *name, char *valuep)
+{
+
+    WT_ASSERT_ALWAYS(session, !F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS),
+      "Cleaning the metadata string should only be called for non-live restore file systems");
+
+    if (strstr(valuep, "live_restore=") != NULL) {
```

Comment:
This is fairly analagous to what we do when we parse the original metadata string to produce the WT_LIVE_RESTORE_FH_META. I don't want to hold this PR up on this request but could we wind the two together? You're sort of hand writing what the WiredTiger config functions were intended to do anyway. 

If that feels like a big ask lets drop a fix me in.

Reply:
Give it a try before the fixme IMO.

Reply:
I think there's enough difference between this and the `meta_ckpt.c` instance that they can stay separate. 
Let me know if you disagree and I'll throw in the FIXME

Reply:
Do we want to #define these config strings we keep dropping around the place? Currently in meta_ckpt.c, this file, and conn_dhandle.c.

I am okay with keeping them separate.